### PR TITLE
Create a GCS bucket for CNI artifacts

### PR DESCRIPTION
**Pull request by thockin**
New group

Minor refactoring of GCP infra (needs more)

New GCS bucket

### CODE REVIEW

**Comment by thockin** [comment]
@dcbw to recommend other owners for this role

File: groups/groups.yaml [hunk 1]
```diff
@@ -287,6 +287,17 @@ groups:
...
       - thockin@google.com
 
+  - email-id: k8s-infra-push-cni@kubernetes.io
+    name: k8s-infra-push-cni
+    description: |-
+      ACL for pushing CNI artifacts
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - saugustus@vmware.com
```
Comment (justaugustus) [inline groups/groups.yaml]:
```suggestion
      - stephen.k8s@agst.us
```

**Comment by thockin** [comment]
Fixed

**Pull request merged**